ui/components/Metadata.tsx

```diff
@@ -1,22 +1,35 @@
 import * as React from "react";
 import styled from "styled-components";
-import Text from "./Text";
+import { isHTTP } from "../lib/utils";
 import Flex from "./Flex";
 import InfoList from "./InfoList";
+import Link from "./Link";
+import Text from "./Text";
 
 type Props = {
   className?: string;
   metadata: any;
-}
+};
 
 function Metadata({ metadata, className }: Props) {
   if (!metadata?.length) {
-    return (<></>);
+    return <></>;
   }
 
+  metadata.forEach((pair) => {
+    if (isHTTP(pair[1]))
+      pair[1] = (
+        <Link newTab href={pair[1]}>
+          {pair[1]}
```

Comment:
It might be easier to read/more efficient to assign `pair[1]` to a constant in the three places where you are just reading the value instead of accessing the element by index.

Reply:
good idea. will do.

Reply:
WAIT but I have to reassign it - If I set it to a variable I'd just be changing the variable not the actual data right?

Reply:
I mean smth. like this:

```
metadata.forEach((pair) => {
    const val = pair[1];

    if (isHTTP(val))
      pair[1] = (
        <Link newTab href={val}>
          {val}
        </Link>
      );
  });
```